src/ctia/schemas/graphql/resolvers.clj

```diff
@@ -75,7 +76,7 @@
 (s/defn indicator-by-id
   [id :- s/Str
    ident]
-  (some-> (read-store :indicator read-indicator id ident)
+  (some-> (read-store :indicator read-indicator id ident {})
```

Comment:
Why is the field selection not used here?

Reply:
I updated those too in my latest commit